src/rimage.c

```diff
@@ -31,6 +31,7 @@ static void usage(char *name)
 	fprintf(stdout, "\t -e build extended manifest\n");
 	fprintf(stdout, "\t -y verify signed file\n");
 	fprintf(stdout, "\t -q resign binary\n");
+	fprintf(stdout, "\t -p set production bit\n");
```

Comment:
please rename to pv bit

Reply:
you mean the description - "set the PV bit?"

Reply:
yeap, variables too, everywhere just PV